File: run-python-with-the-snapchat-marketing-api-p_jmC6LmD/code/entry.py
import requests

def handler(pd: "pipedream"):
  token = f'{pd.inputs["snapchat_marketing"]["$auth"]["oauth_access_token"]}'
  authorization = f'Bearer {token}'
  headers = {"Authorization": authorization}
  r = requests.get('https://adsapi.snapchat.com/v1/me', headers=headers)
  # Export the data for use in future steps
  return r.json()

File: run-node-js-with-the-snapchat-marketing-api-p_RRC2252/code/entry.js
import { axios } from "@pipedream/platform"
export default defineComponent({
  props: {
    snapchat_marketing: {
      type: "app",
      app: "snapchat_marketing",
    }
  },
  async run({steps, $}) {
    return await axios($, {
      url: `https://adsapi.snapchat.com/v1/me`,
      headers: {
        Authorization: `Bearer ${this.snapchat_marketing.$auth.oauth_access_token}`,
      },
    })
  },
})
